src/main/java/Duke.java

```diff
@@ -1,10 +1,33 @@
+import java.util.Scanner;
+
```

Comment:
Should this blank line be removed in order to collect all the import statements together?

Reply:
Resolved. Thank you!